sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -204,6 +201,17 @@ def _get_severity_level(severity_number: Optional[SeverityNumber]):
         return 0
     return int((severity_number.value - 1) / 4 - 1)
 
+def _body_to_string(log_body: Any) -> str:
+    if not log_body:
+        return "n/a"
+
+    if isinstance(log_body, str):
+        return log_body[:32768]
+
+    if isinstance(log_body, Exception):
```

Comment:
Nit: We don't have to handle `Exception` case specifically

Reply:
would you rather `json.dump` it too? 

```log
>>> import json
>>> ex = Exception("123")
>>> json.dumps(ex)

TypeError: Object of type Exception is not JSON serializable
```

it's probably awful for perf and best to prevent it. Also this helps to keep things backward-compatible for exceptions.